[2025-06-27T06:17:24.620+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T06:17:24.736+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_dev.start_task scheduled__2025-01-05T09:00:00+00:00 [queued]>
[2025-06-27T06:17:24.863+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0_dev.start_task scheduled__2025-01-05T09:00:00+00:00 [queued]>
[2025-06-27T06:17:24.881+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T06:17:24.932+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): start_task> on 2025-01-05 09:00:00+00:00
[2025-06-27T06:17:25.179+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4057) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T06:17:25.201+0000] {standard_task_runner.py:72} INFO - Started process 4088 to run task
[2025-06-27T06:17:25.212+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0_dev', 'start_task', 'scheduled__2025-01-05T09:00:00+00:00', '--job-id', '1006', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpqdrgyd_c']
[2025-06-27T06:17:25.234+0000] {standard_task_runner.py:105} INFO - Job 1006: Subtask start_task
[2025-06-27T06:17:25.532+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0_dev.start_task scheduled__2025-01-05T09:00:00+00:00 [running]> on host 546ad4141aeb
[2025-06-27T06:17:25.877+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0_dev' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-05T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-05T09:00:00+00:00'
[2025-06-27T06:17:25.894+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T06:17:25.939+0000] {logging_mixin.py:190} INFO - start
[2025-06-27T06:17:25.941+0000] {logging_mixin.py:190} INFO - ds: 2025-01-05
[2025-06-27T06:17:25.945+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-27T06:17:25.980+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T06:17:25.983+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0_dev, task_id=start_task, run_id=scheduled__2025-01-05T09:00:00+00:00, execution_date=20250105T090000, start_date=20250627T061724, end_date=20250627T061725
[2025-06-27T06:17:26.021+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T06:17:26.207+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T06:17:26.242+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
